charts/kubefed/values.yaml

```
@@ -18,7 +18,10 @@ controllermanager:
   ## Supported options are `configmaps` and `endpoints`
   leaderElectResourceLock:
   syncController:
+    maxConcurrentReconciles:
     adoptResources:
```

nit (not a blocker): would be good to sort these alphabetically I think but I don't mind leaving as is
```suggestion
    adoptResources:
    maxConcurrentReconciles:
```

---

Just want to put the general options above the specific options, in accordance of the type defination of KubeFedConfig. And other options in this file are also not sorted alphabetically, so I'm going to leave it as is.

---

:+1: I'm down with that :)